radial_hit.m

```diff
@@ -15,38 +9,38 @@
 %    circle_max_radius: The max radius of the circle.
 %    delta_radius: The delta of the radial sections.
 %    t: The current time parameter of the ray.
-%    ray_unit_vector: The ray unit vector.
-%    ray_circle_vector: The vector difference between the ray origin and the circle origin.
-%    v: The dot product between the ray_unit_vector and the ray_circle_vector.
-%    prev_transition_flag: Determines whether the previous radial traversal
-%    was a 'transition.' A transition is defined as the turning point from
-%    inward movement to outward movement from the circle's origin. Another
-%    way this can be denoted is sequential hits with equal radii.
 %    verbose: Determines whether debug print statements are enabled.
 %
 % Returns:
 %    tMaxR: is the time at which a hit occurs for the ray at the next point of intersection.
-%    tStepR: The voxel traversal value: 0, +1, -1.
-%    transition_flag: Determines whether the current voxel traversal was a
-%    'transition.'
 
 if verbose
     fprintf('\n--radial_hit-- \nCurrent Radial Voxel: %d', current_radial_voxel)
 end
 
+
 % Recapture the current radius of ray.
 r = circle_max_radius - delta_radius * (current_radial_voxel - 1);
 
-r_a = max(r - delta_radius , delta_radius);
+% Check for intersections with relevant radial neighbors.
 
+% TODO: ray_unit_vector, ray_circle_vector, v can be calculated during
```

Comment:
You approved my PR to calculate these outside of radial_hit, is there a reason why you put them back?